### PR TITLE
test cases added

### PR DESCRIPTION
**Pull request by HarithaIBM**

### CODE REVIEW

File: buildenv [hunk 1]
```diff
@@ -42,5 +36,5 @@ zopen_get_version()
 {
   # Modify to echo the version of your tool/library
   # Rather than hardcoding the version, obtain the version by running the tool/library
-  echo "1.0.0"
+  echo "2.21"
```
Comment (IgorTodorovskiIBM) [inline buildenv]: Maybe you can put 2.21 into a global variable or extract the version from the which command

**Review by MikeFultonDev — APPROVED** [review]
approved pending igor's suggestion

File: buildenv (hunk 1 above)
Comment (HarithaIBM) [reply buildenv]: I have modified this, thank you.

File: buildenv [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
 export ZOPEN_TARBALL_DEPS="curl gzip make m4 grep sed gettext coreutils"
-export ZOPEN_GIT_DEPS="curl gzip make m4 grep sed gettext coreutils automake autoconf zoslib"
 export ZOPEN_TYPE="TARBALL"
-#export ZOPEN_TYPE="GIT"
-#export ZOPEN_BOOTSTRAP="./autogen.sh"
+export __UNTAGGED_READ_MODE=ASCII
+export ZOPEN_CHECK="./which"
+export ZOPEN_CHECK_OPTS="--version"
 
+WHICH_VERSION=$(which --version | cut -d "," -f 1 | head -n 1 | cut -d " " -f 3 | tr -d 'v')
```
Comment (IgorTodorovskiIBM) [inline buildenv]: Is which in the PATH when buildenv is loaded? Might be better to put this in zopen_get_version

Comment (HarithaIBM) [reply buildenv]: I have modified it to fetch the value in zopen_get_version and zopen_check_results, thank you

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Pull request merged**